packages/cozy-harvest-lib/src/components/AccountSelectBox/AccountSelectorHeader.tsx

```diff
@@ -9,9 +9,13 @@ import Typography from 'cozy-ui/transpiled/react/Typography'
 import AccountSelectBox from './AccountSelectBox'
 import { useDialogContext } from '../DialogContext'
 
-interface AccountSelectorHeaderProps {
+export interface AccountSelectorHeaderProps {
   konnector: { slug: string }
-  account: Record<string, unknown> & { _id: string }
+  account: Record<string, unknown> & {
+    _id: string
+    account_type: string
+    auth: { login: string }
```

Comment:
You can use `IOCozyAccount` from cozy-client ;) https://unpkg.com/browse/cozy-client@38.8.0/types/types.d.ts